tests/operations/unary/RandomDraw.cpp

```diff
@@ -1,11 +1,69 @@
 #define CATCH_CONFIG_MAIN
 #include "Catch/single_include/catch2/catch.hpp"
 
+#include "sgpl/algorithm/execute_core.hpp"
```

Comment:
Instead of testing the specific way that it's implemented (which might change) we should be testing the /behavior/.